iree/compiler/Conversion/LinalgToLLVM/MarkerUtils.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Google LLC
```

Comment:
Based on comment above this file could be removed by making MarkerUtils.h/MarkerUtils.cpp a common file between CPU and GPU